frontend/src/constants/reducerTypes.ts

```diff
@@ -7,7 +7,7 @@ export const PARCEL = 'parcel';
 export const LOOKUP_CODE = 'lookupCode';
 
 // access requests
-export const ACCESS_REQUEST = 'accessRequest';
+export const ACCESS_REQUEST = 'accessRequests';
```

Comment:
May need another solution for this. The redux repo advocates for just naming slices with a string and then using that exact string when setting up each reducer in the store. This doesn't seem as safe as using a static string in both places.

Reply:
Redux toolkit does this in a much cleaner way.  Removing the need to maintain these odd little files with strings.  Just move the value directly into your slice.